packages/ui/src/components/Stepper/index.tsx

```diff
@@ -8,6 +8,15 @@ import { Text } from '../Text'
 
 type Temporal = 'previous' | 'next' | 'current'
 
+const SIZES = {
```

Comment:
```suggestion
const LINE_HEIGHT_SIZES = {
```

better to understand the usage ?